### DISCUSSION TITLE
Is rank available in Lazer?

### DISCUSSION THREAD

**Discussion by BojayB** [ppy/osu#13291] [Q&A] [answered]
I've been playing on Lazer more often recently because I use Linux on my computer. I've realized recently that I haven't gained any rank whatsoever even after playing a map I haven't before, nor does my play count increase. Hell, it doesn't even say if I'm online! I know PP Isn't implemented yet or won't get implemented, but is the rest a mistake or is it coming soon?

My rank just keeps dropping and dropping, it's destroying my ego man!

**Comment by Game4all** [comment, marked as answer]
Your play statistics don't get updated when you play lazer for the moment being; same applies for scores achieved on lazer which aren't submitted to bancho. You'll have to play on osu!stable to gain ranks and pp for the moment being.

**Reply by BojayB** [reply]
man that sucks. osu!stable doesn't work properly through wine and it's time consuming to reboot whenever I feel like using windows to play osu!. Any idea as to when Lazer will update to bancho? Or is it coming SOON™

**Reply by peppy** [reply]
"bancho" is a chat and presence server, which is not intended to ever be used by lazer.

if you mean "when are my statistics going to update", then that is on our roadmap as one of the next things to happen. as you may expect, we are treading cautiously and making sure everything is in a good state as to not break years of consistency in scoring / statistics etc.
